lib/lndclient/LndClient.ts

```diff
@@ -506,20 +506,28 @@ class LndClient extends SwapClient {
   private executeSendRequest = async (
     request: lndrpc.SendRequest,
   ): Promise<string> => {
+    if (!this.isConnected()) {
+      throw swapErrors.FINAL_PAYMENT_ERROR(errors.UNAVAILABLE(this.currency, this.status).message);
+    }
+
     this.logger.trace(`sending payment with ${JSON.stringify(request.toObject())}`);
     let sendPaymentResponse: lndrpc.SendResponse;
     try {
       sendPaymentResponse = await this.sendPaymentSync(request);
     } catch (err) {
       this.logger.error('got exception from sendPaymentSync', err);
-      throw swapErrors.PAYMENT_ERROR(err.message);
+      if (typeof err.message === 'string' && err.message.includes('chain backend is still syncing')) {
+        throw swapErrors.FINAL_PAYMENT_ERROR(err.message);
```

Comment:
Same here. Shouldn't we retry until LND is operational again?

Reply:
In this case lnd is operational, but it is won't accept send payment calls because it is out of sync. This is also an edge case as we wait for lnd to sync to chain before we'll attempt any swaps with it, but I just put this here to cover the edge case where it goes back out of sync somehow.